plugins/package-managers/node/src/main/kotlin/Yarn.kt

```diff
@@ -88,9 +87,9 @@ class Yarn(
 
         val process = run(workingDir, "info", "--json", packageName)
 
-        val data = Json.parseToJsonElement(process.stdout).jsonObject["data"]?.also {
+        val data = parseJsonLines(process.stdout).jsonObject["data"]?.also {
             yarnInfoCache.write(packageName, it.toString())
-        } ?: checkNotNull(Json.parseToJsonElement(process.stderr).jsonObject["data"]).also {
+        } ?: checkNotNull(parseJsonLines(process.stderr).jsonObject["data"]).also {
```

Comment:
I believe we now have two function which have the responsibility of extracting the packageJson content
from yarns output. This one here, plus the `parseJsonLines`. To improve cohesion, I believe it would be better to have a single function doing it completely, instead of two functions doing this partially. Wht do you think?

(Maybe such function should remain under the `Yarn` umbrella, not under the `PackageJson` one.